package/kedro_viz/launchers/cli/run.py

```diff
@@ -81,7 +81,7 @@
 @click.option(
     "--lite",
     is_flag=True,
-    help="A flag to load an experimental light-weight Kedro Viz",
+    help="An experimental flag to open Kedro-Viz with missing Kedro project dependencies",
```

Comment:
```suggestion
    help="Run `kedro-viz` in lightweight mode without requiring all project dependencies.",
```

Reply:
So it's no longer "experimental"? 😄 

(Not that I'm pressing for it to be included again - we never defined what "experimental" means anyway...)

Reply:
I was thinking from the sense of -- does it help the user?

Reply:
I agree we do not define what experimental means. I thought this would be something the user will proceed with caution considering the limitations if we tag it as experimental. Happy to discard as well. Also the lightweight mode is too wordy to understand for the user. Can we have - 
```
help="A flag to run Kedro-Viz without Kedro project dependencies"
```

Reply:
help="An experimental flag to run Kedro-Viz without Kedro project dependencies"

I think you both are right, we treat `kedro-viz --lite` as experimental. Once we see the feature's success, we can integrate it into our normal workflow and remove the `--lite`. Since this feature may not be permanent, adding it as an experimental option seems like a good idea.